ci/build.ps1

```diff
@@ -97,15 +97,21 @@ bin\nvim --version ; exitIfFailed
 
 # Functional tests
 # The $LastExitCode from MSBuild can't be trusted
-$failed = $false
```

Comment:
I don't think this is a problem because the MSBuild workaround should work with both MSVC and MinGW builds. The error message that it's searching for, `Running functional tests failed with error`, comes from `cmake/RunTests.cmake` and it should be printed when a test fails regardless of which generator is being used.

Reply:
See the MINGW_32 log at https://ci.appveyor.com/project/neovim/neovim/build/19862/job/y452uy85m5fsl7rl
The build succeeded but it failed in the functional tests.

Reply:
https://ci.appveyor.com/project/neovim/neovim/build/19866/job/2vpjs501eti96t5b
should have failed in functional tests but it ran the oldtests anyway.

Reply:
Yes, adding `2>&1` before the `|` should make it work. It appears that the error message isn't printed to stdout for the MinGW builds, so it wasn't being detected.